env/build_noaacloud_gnu.env

```diff
@@ -0,0 +1,17 @@
+#environment for building on NOAA-Cloud
+module purge
+module use /contrib/Mark.Potts/spack/share/spack/modules/linux-centos7-haswell
+module use /contrib/Mark.Potts/spack/share/spack/modules/linux-amzn2-skylake_avx512
+module use /contrib/EPIC/hpc-modules/modulefiles/stack/
+module load gcc/9.3.0
+module load openmpi
+module load hpc hpc-gcc hpc-openmpi
+module use /apps/modules/modulefiles 
+module load rocoto
+module load netcdf/4.7.4 crtm/2.3.0 nemsio/2.5.4 pio/2.5.3 upp/10.0.9 nemsiogfs/2.5.3 esmf/8_2_0_beta_snapshot_14 gfsio/1.4.1 wgrib2/2.0.8 cmake/3.20.1 g2/3.4.5 sfcio/1.4.1 jasper/2.0.25 g2tmpl/1.10.0 sigio/2.3.2 fms/2021.03 zlib/1.2.11 ip/3.3.3 landsfcutil/2.4.1 png/1.6.35 sp/2.3.3 bacio/2.4.1 hdf5/1.10.6 w3nco/2.4.1 w3emc/2.9.2 miniconda3
```

Comment:
It is often easier to manage module lists if each module appears on a single line.

Reply:
I'll make those chnages.